test/mux.sh

```diff
@@ -0,0 +1,353 @@
+#!/bin/bash
+
+# test the mux route and service
+# - can accept secure_forward from a "client" fluentd
+
+if [ -z "${MUX_CLIENT_MODE:-}" -o "${MUX_ALLOW_EXTERNAL:-false}" = "false" ]; then
+    echo "Skipping -- This test requires MUX_CLIENT_MODE and MUX_ALLOW_EXTERNAL are true."
+    exit 0
+fi
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/deployer/scripts/util.sh"
+os::util::environment::use_sudo
+
+os::test::junit::declare_suite_start "test/mux"
+
+if [ -n "${DEBUG:-}" ] ; then
+    set -x
+fi
+
+reset_fluentd_daemonset() {
+  # this test only works with MUX_CLIENT_MODE=minimal for now
+  os::log::debug "$( oc set env daemonset/logging-fluentd MUX_CLIENT_MODE=minimal )"
```

Comment:
I would suggest you use `os::cmd::expect_success` then -- it will collapse the output into one or two lines and give nice output on failures. You'll get jUnit too, which makes programmatic identification of issues easier.